python/torch_mlir_e2e_test/test_suite/scalar.py

```diff
@@ -127,6 +127,29 @@ def DivFloatModule_basic(module, tu: TestUtils):
 # ==============================================================================
 
 
+class DivIntModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([], torch.float32, True),
+        ([], torch.float32, True),
+    ])
+    def forward(self, lhs, rhs):
+        return lhs / rhs
```

Comment:
nit: can you use `torch.ops.aten.div`, to make it clear what is being tested? Also, you should be casting the arguments to ints to make sure you're not testing the `div.Tensor` variant, i.e.

```python
     return torch.ops.aten.div(int(lhs), int(rhs))
```